src/quantum/index-32bit.js

```diff
@@ -108,326 +109,39 @@ export default class QuantumIndex32 extends React.Component {
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tabpaint',
-                  },
-                },
-              ]}
-            />,
-            <PerfherderGraphContainer
-              title='Tab Animation (TART)'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    extraOptions: ['e10s', 'stylo'],
-                    frameworkId: 1,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tart',
-                  },
-                },
-              ]}
-            />,
-          ],
-          [
-            <PerfherderGraphContainer
-              title='Tab Switch (tps)'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    extraOptions: ['e10s', 'stylo'],
-                    frameworkId: 1,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tps',
-                  },
-                },
-              ]}
-            />,
-            <PerfherderGraphContainer
-              title='SVG (tsvg_static)'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    extraOptions: ['e10s', 'stylo'],
-                    frameworkId: 1,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tsvg_static',
-                  },
-                },
-              ]}
-            />,
-          ],
-          [
-            <PerfherderGraphContainer
-              title='SVG (tsvgr_opacity)'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    extraOptions: ['e10s', 'stylo'],
-                    frameworkId: 1,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tsvgr_opacity',
-                  },
-                },
-              ]}
-            />,
-            <PerfherderGraphContainer
-              title='SVG (tsvgx)'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    extraOptions: ['e10s', 'stylo'],
-                    frameworkId: 1,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'tsvgx',
-                  },
-                },
-              ]}
-            />,
-          ],
-        ],
+        title: '#3 Pages',
```

Comment:
Why the notion of pages?
It's only the tp6 jobs that load pages. All the other benchmarks do other things than loading pages.
Please use 'benchmarks'.

Reply:
OK